docs/developer-docs/devquests.md

```diff
@@ -408,6 +408,25 @@ You must have made sets for at least 15 subgenres before opt-in. To opt-in, make
 4. Homebrew that is essentially the same as another you've done will not count (i.e. multiple Flappy Bird clones can only garner credit once)
 5. One set per large homebrew collection maximum (i.e. One Net Yaroze 2014 Collection set allowed)
 
+### 22 [This Belonged in a Museum!](https://retroachievements.org/game/25678)
+
+![DQ022](https://retroachievements.org/Images/102134.png)
+
+**Type**: Revision, Maintenance 
+**Goal**: Obtain 4 points by dusting off older sets, bringing them up to current RA standards  
+**Repeatable**: No
+
+1. The Quest is completed upon obtaining 4 points towards it. Full Set Dust-Off = 1 Point, Misc Task = 0,25pt, at least one set dust-off required for the badge.
```

Comment:
Probably should spell it out as 0.25 points (American-centric, I know), or as "a quarter point." Or use the 1/4 Unicode character.

Reply:
Per internal DQ Team talk, updating the point framework to require 12 Total, meaning the dust-off is now worth 4pts and the misc 1pt each.